routes/index.go

```diff
@@ -108,12 +107,11 @@ func NewRouter() *http.Server {
 	server := &http.Server{Addr: ":" + PORT, Handler: r}
 
 	go func() {
-		fmt.Println("Listening on port " + PORT)
+		utils.Log.Info("Listening on port %s", PORT)
```

Comment:
`PORT` is a digit so you'll need to use `%d`

also since we are going from `fmt.Println` to `log.Printf` so you'll need to add `\n` at the end of the log

Reply:
nvm it does create a new line in the logger but we still may need to use %d

Reply:
@kevkevinpal, Actually, since PORT is a string in the code (from os.Getenv("PORT")), we should keep using %s. What's your opinion?

Reply:
I'm wrong about that aswell seems like we're treating PORT as a string